sklearn/metrics/tests/test_common.py

```diff
@@ -647,42 +645,20 @@ def test_multioutput_regression_invariance_to_dimension_shuffling():
                                         "invariant" % name)
 
 
-@ignore_warnings
 def test_multilabel_representation_invariance():
     # Generate some data
     n_classes = 4
     n_samples = 50
-    # using sequence of sequences is deprecated, but still tested
-    make_ml = ignore_warnings(make_multilabel_classification)
-    _, y1 = make_ml(n_features=1, n_classes=n_classes, random_state=0,
-                    n_samples=n_samples)
-    _, y2 = make_ml(n_features=1, n_classes=n_classes, random_state=1,
-                    n_samples=n_samples)
-
-    # Be sure to have at least one empty label
-    y1 += ([], )
-    y2 += ([], )
```

Comment:
I manually checked for both `random_state` of 0 and 1 and found it had an empty label...

But indeed explicitly adding would be better! I'll add one...